examples/CSharp/OpenGL/Tutorial 1.4 - Textures/BufferObject.cs

```diff
@@ -0,0 +1,36 @@
+using Silk.NET.OpenGL;
+using System;
+
+namespace Tutorial
+{
+    public class BufferObject<TDataType> : IDisposable
+        where TDataType : unmanaged
+    {
+        private uint _handle;

```

Comment:
I see no reason why these can't be public get-only properties. Only a minor change though - your call.